inc/template.php

```diff
@@ -1864,6 +1866,25 @@ function tpl_classes() {
         (isset($INFO) && $INFO['exists']) ? '' : 'notFound',
         ($ID == $conf['start']) ? 'home' : '',
     );
+    $id=$ID;
+    if($id[0] == ':')
+        $id = substr($id,1);
+    $id = str_replace('_', '__', $id);
+    $chain = explode(':', $id);
+    $pgname = array_pop($chain);
+
+    $classes[] = 'dwl_' . count($chain);
+    $ns = 'dwn_';
+    foreach($chain as $comp) {
+        $ns .= "_$comp";
+        $classes[] = $ns;
+    }
+    $parent = array_pop($chain);
+    $classes[] = "dwn_{$parent}_";
+    $classes[] = 'dwp_'.$pgname;
```

Comment:
nit: consistent style

```suggestion
    $classes[] = "dwp_$pgname";
```